packages/webapp/src/stories/TileDashboard/TileDashboard.stories.tsx

```diff
@@ -0,0 +1,179 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { ReactNode } from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+import styles from './styles.module.scss';
+import { Main } from '../../components/Typography';
+import { PureTileDashboard } from '../../components/TileDashboard';
+import { componentDecorators } from '../Pages/config/Decorators';
+import { ReactComponent as AlpacaIcon } from '../../assets/images/animals/alpaca-icon-btn-list.svg';
+import { ReactComponent as CattleIcon } from '../../assets/images/animals/cattle-icon-btn-list.svg';
+import { ReactComponent as ChickenIcon } from '../../assets/images/animals/chicken-icon-btn-list.svg';
+import { ReactComponent as GoatIcon } from '../../assets/images/animals/goat-icon-btn-list.svg';
+import { ReactComponent as PigIcon } from '../../assets/images/animals/pig-icon-btn-list.svg';
+import { ReactComponent as RabbitIcon } from '../../assets/images/animals/rabbit-icon-btn-list.svg';
+import { ReactComponent as SheepIcon } from '../../assets/images/animals/sheep-icon-btn-list.svg';
+
+// https://storybook.js.org/docs/writing-stories/typescript
+const meta: Meta<typeof PureTileDashboard> = {
+  title: 'Components/PureTileDashboard',
```

Comment:
Could we call this just `TileDashboard`? I think the "Pure" makes it harder for me to search for what I'm looking for

Reply:
Sure -- rename just the Storybook story you mean (as opposed to the component + Story?)

I already get thrown off by how the component folder name (doesn't include Pure) and the export name (includes Pure) don't match, but I thought there was a reason behind that pattern so I kept following it. Definitely seems pointless in Storybook though where they're supposed to all be pure components.

Reply:
Yeah that's what I meant! Just the story 🙂